Libraries/Image/ImageEditor.js

```diff
@@ -32,14 +32,15 @@ type ImageCropData = {
   },
   /**
    * (Optional) size to scale the cropped image to.
+   * defaults to size. (no scale)
```

Comment:
Do the docs explain anywhere what the options are? It doesn't look like it, to me. Could you just add a brief explanation of what the valid values are and what they do? Otherwise it doesn't make much sense to say the default is "contain".